app/components/sessions/sessions.html

```diff
@@ -13,7 +13,7 @@
 
         <md-content style="height: 720px">
             <section ng-repeat="day in sc.Days" ng-if="day.sessions.length > 0">
-                <md-subheader class="md-primary"> {{ day.label }}</md-subheader>
+                <md-subheader style="margin-right:35px;" class="md-primary"> {{ day.label }}</md-subheader>
```

Comment:
I don't think its a good idea to inline the CSS. It's probably better if this is done in CSS / SCSS files.